src/pages/[platform]/build-a-backend/data/aws-appsync-apollo-extensions/index.mdx

```diff
@@ -266,62 +376,85 @@ func createApolloClient() throws -> ApolloClient {
 
-If you define multiple authorization strategies on a single model, you will have to create separate Apollo client instances for each Authorizer that you want to use in your app.
+If you define multiple authorization strategies within your schema, you will have to create separate Apollo client instances for each Authorizer that you want to use in your app.
+
+</InlineFilter>
 
-### Downloading the AWS AppSync schema
+## Downloading the AWS AppSync schema
 
 The schema is used by Apollo’s code generation tool to generate API code that helps you execute GraphQL operations. The following steps integrate your AppSync schema with Apollo's code generation process:
 
 <InlineFilter filters={["swift"]}>
 1. Navigate to your API on the [AWS AppSync console](https://console.aws.amazon.com/appsync/home)
 2. On the left side, select Schema
-3. When viewing your schema, there should a “Export schema” drop down. Select this and download the `schema.json` file.
+3. Select the "Export schema" dropdown and download the `schema.json` file.
 4. Add this file to your project as directed by [Apollo Code Generation documentation](https://www.apollographql.com/docs/ios/code-generation/introduction). 
 
 You can alternatively download the introspection schema using the [`fetch-schema`](https://www.apollographql.com/docs/ios/code-generation/codegen-cli#fetch-schema) command with the `amplify-ios-cli` tool.
 
 </InlineFilter>
 
 <InlineFilter filters={["android"]}>
+
 1. Navigate to your API on the [AWS AppSync console](https://console.aws.amazon.com/appsync/home)
 2. On the left side, select Schema
-3. When viewing your schema, there should a “Export schema” drop down. Select this and download the `schema.json` file.
+3. Select the "Export schema" dropdown and download the `schema.json` file.
 4. Add this file to your project as directed by [Apollo documentation](https://www.apollographql.com/docs/kotlin/advanced/plugin-recipes#specifying-the-schema-location) 
 
-### Type Mapping AppSync Scalars
+</InlineFilter>
+
+## Generating Queries, Mutations, and Subscriptions for Apollo client
+
+<InlineFilter filters={["android"]}>
+
+<BlockSwitcher>
+
+<Block name="With Amplify">
+### Amplify provided .graphql files
+1. Within your Amplify Gen2 backend, run: `npx ampx generate graphql-client-code --format graphql-codegen --statement-target graphql --out graphql`
+2. Copy the generated files (`mutations.graphql`, `queries.graphql`, `subscriptions.graphql`) to your `{app}/src/main/graphql` folder as shown in the [Apollo documentation](https://www.apollographql.com/docs/kotlin#getting-started)
+
+### Manual
+1. Navigate to the **Queries** tab in your API on the [AWS AppSync console](https://console.aws.amazon.com/appsync/home). Here, you can test queries, mutations, and subscriptions in the GraphQL playground.
+2. Enter your GraphQL operation (query, mutation, or subscription) in the editor and click **Run** to execute it.
```

Comment:
```suggestion
2. Enter your GraphQL operation (query, mutation, or subscription) in the editor and select **Run** to execute it.
```

nit "click" -> "select"
https://github.com/aws-amplify/docs/blob/main/STYLEGUIDE.md#15-use-carets-to-help-users-navigate-menus-use-select-instead-of-click